man-roxygen/show_in_console.R

```diff
@@ -1,2 +1,2 @@
 #' @param show_in_console Show output in the R console? If FALSE,
-#' then it is saved to a file.
+#' then it is saved to a file at the end of the model run.
```

Comment:
Do you  need to have this in a template given that it is only used in a single function, i.e., `run()`. Also, there is no default argument called out in the explanation. Line two should be indented by two spaces. FALSE should be wrapped in back ticks. It is not 100% clear what "it" on line two is.